Earthfile

```diff
@@ -226,6 +227,9 @@ docker:
         COPY overlay/files-opensuse-arm-rpi/ /
     ELSE IF [ "$FLAVOR" = "opensuse-arm-rpi" ]
         COPY overlay/files-opensuse-arm-rpi/ /
+    ELSE IF [ "$FLAVOR" = "ubuntu" ]
+        COPY overlay/files-ubuntu/ /
+        COPY overlay/k8s/$K8S_FLAVOR/ /
```

Comment:
same, the overlay files here are very specific to the kubernetes engines - those changes are not meant to be here

Reply:
I will remove these.